go.mod

```diff
@@ -34,6 +34,8 @@ require (
 	k8s.io/kubernetes v1.23.0
 )
 
+require github.com/stretchr/testify v1.7.0
```

Comment:
same as your other PR .... either put this with the other requires, or put a comment explaining why it is separated

Reply:
updated.